lib/android/src/main/java/com/airbnb/android/react/maps/AirMapManager.java

```diff
@@ -332,14 +345,15 @@ public Map getExportedCustomDirectEventTypeConstants() {
   @Nullable
   @Override
   public Map<String, Integer> getCommandsMap() {
-    Map<String, Integer> map = MapBuilder.of(
+    Map<String, Integer> map = this.CreateMap(
```

Comment:
Why this? `CreateMap` looks to me now very unreadable and limiting (only 8 key/values pairs allowed and ambiguous naming ->  `k1,v1`)